content/en/docs/Reference/catalog-templates.md

```diff
@@ -503,3 +503,357 @@ schema: olm.channel
+
+### Usage
+```sh
+opm alpha render-template composite [flags] <filename>
+```
+
+| Flag                | Description                                                                            |
+| ------------------- | -------------------------------------------------------------------------------------- |
+| -h, --help          | prints help/usage information                                                          |
+| -o, --output <type> | the output format, can be `yaml` or `json`                                             |
+| -c, --composite-config <file> | File to use as the composite configuration file (default "catalog/config.yaml") |
+| --skip-tls-verify   | skip TLS certificate verification for container image registries while pulling bundles |
+| --use-http          | use plain HTTP for container image registries while pulling bundles                    |
+| --validate          | whether or not the created FBC should be validated (i.e 'opm validate') (default true) |
+| --container-tool <tool> | container tool to be used when rendering veneers (should be an equivalent replacement to docker - similar to podman) (default "docker") |
+
+## Specification
+The composite veneer consists of the following schemas:
+- `olm.composite`
+    - This schema defines what, where, and how to build a particular contribution
+- `olm.composite.catalogs`
+    - This schema defines a catalog and the constraints for building the catalog
+
+The `olm.composite.catalogs` cue schema is:
+```cue
+#CatalogConfig: {
+	Schema: "olm.composite.catalogs"
+	Catalogs: [... #Catalog]
+}
+
+#Catalog: {
+	Name:        string
+	Destination: #CatalogDestination
+	Builders: [...string]
+}
+
+#CatalogDestination: {
+	BaseImage:  string
+	WorkingDir: string
+}
+```
+
+The `olm.composite` cue schema is:
+```
+#CompositeConfig: {
+	Schema: "olm.composite"
+	Components: [...#Component]
+}
+
+#Component: {
+	Name:        string
+	Destination: #ComponentDestination
+	Strategy:  #BuildStrategy
+}
+
+#ComponentDestination {
+	Path: string
+}
+
+#BuildStrategy: {
+	Name:   string
+	Template: #TemplateDefinition
+}
+
+#TemplateDefinition: {
+	Schema: string
+	Config: _ @go(,json.RawMessage)
+}
+```
+
+### Examples
+**Catalog Config File**
+
+The following example creates a catalog configuration that:
+- Defines the `v4.10` catalog and restricts contributions to the catalog to only being `basic` templates
+- Defines the `v4.11` catalog and restricts contributions to the catalog to only being `semver` templates
+- Defines the `v4.12` catalog and restricts contributions to the catalog to only being raw FBC
```

Comment:
Internal to the template, we call this 'raw FBC', but to any audience outside the tool this is just 'FBC'.  Perhaps if we add a section in specifications which lists the supported builder types, this can just be a reference to that builder.